common/reviews/api/rush-lib.api.ts

```diff
@@ -264,6 +268,17 @@ module RushConstants {
 
 }
 
+// (undocumented)
```

Comment:
You also need @public

---
In reply to: [116885162](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r116885162) [](ancestors = 116885162,116885084)